internal/testhelper/testhelper.go

```diff
@@ -65,118 +47,6 @@ type Service interface {
 	GetSLAMProcessBufferedLogReader() bufio.Reader
 }
 
-// SetupDeps returns the dependencies based on the sensors passed as arguments.
-func SetupDeps(sensors []string) resource.Dependencies {
```

Comment:
I needed to be able to use this test in the sensorprocess package, but couldn't b/c this file is in an internal pacakge. The package is internal b/c other functions in it need to have access to viamcartographer internals.  I noticed that this function didn't depend on viamcartographer internals. As a result, I moved it to a new, non internal testhelper package so it can be used both in viam-cartographer tests & in the sensorprocess tests.